pom.xml

```diff
@@ -81,6 +81,14 @@
             <version>0.4.6</version>
             <scope>test</scope>
         </dependency>
+        <!-- Suppress the Failed to load class org.slf4j.impl.StaticLoggerBinder warning -->
+        <!-- https://www.slf4j.org/codes.html#StaticLoggerBinder -->
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-nop</artifactId>
```

Comment:
This will disable log messages in test execution. I don't think that is a good idea.

Reply:
But we have no logger at all right now, where are these messages supposedly going?